lib/exportJob.js

```diff
@@ -350,9 +378,9 @@ class ExportJob extends EventEmitter {
       }
     } else {
       pageDim = pageDimensions[pageSize]
+      // Flip if landscape
+      pageDim = landscape ? {x: pageDim.y, y: pageDim.x} : pageDim
```

Comment:
Orientation should never be changed if someone specified custom dimensions, the landscape setting will just be ignored in this scenario.  The browser window size and the PDF size ought to be dictated only by the pageSize object.